opencga-app/app/cloud/docker/opencga-init/override_yaml.py

```diff
@@ -9,6 +9,7 @@
 parser.add_argument("--database-prefix", required=False)
 parser.add_argument("--search-hosts", required=True)
 parser.add_argument("--clinical-hosts", required=True)
+parser.add_argument("--rga-hosts", required=True)
```

Comment:
This "required=True" will break the `test_override_yaml.py` . You'll need to check and fix the test.

Reply:
I supposed same value of required than clinical and search hosts.
I change to requierd=False